tests/de.cpp

```diff
@@ -68,21 +68,27 @@ BOOST_AUTO_TEST_CASE(de_evolve_test)
 {
     // Here we only test that evolution is deterministic if the
     // seed is controlled for all variants
-    problem prob1{rosenbrock{25u}};
-    population pop1{prob1, 5u, 23u};
-    problem prob2{rosenbrock{25u}};
-    population pop2{prob2, 5u, 23u};
+    problem prob{rosenbrock{25u}};
```

Comment:
I think this name change bothers appveyor as later some prob is redefined in a different scope .... https://ci.appveyor.com/project/ci4esa/pagmo2/build/1.0.364#L1163